### DISCUSSION TITLE
Nothing pushed back to netbox.

### DISCUSSION THREAD

**Discussion by NEO-AMiGA** [LoH-lu/netbox-nmap-scan#8] [Q&A] [answered]
Hello

I'm testing this out but have ran into some snafu here. 

Everything seems to work fine until I try to `netbox_push.py` back to netbox. There are no errors and it looks fine, but the `Last scan` field is not populated and I can't see that anything have been updated nor added to netbox. 

I do get contents into `ipam_addresses.csv` so it does talk to netbox and it does the scan. Is there any way to troubleshoot the push? With some verbose mode or such?

thanks.

**Comment by LoH-lu** [comment]
Hi @NEO-AMiGA  , can you confirm which version of Netbox you are running ?

**Reply by NEO-AMiGA** [reply]
Hello

NetBox Community v4.1.2

**Reply by LoH-lu** [reply]
Can you confirm as well if you have created properly the custom field scantime ?
![image](https://github.com/user-attachments/assets/42ef3fc3-e3fe-47cf-b004-1b0120145b09)


I will check to make it optional in the future.

**Comment by NEO-AMiGA** [comment]
Yes. I have that. See screenshot. 

<img width="1253" alt="Screenshot 2024-11-04 at 10 42 15" src="https://github.com/user-attachments/assets/ab075cd5-5b07-4d2e-ba66-13ac0703220f">

**Reply by LoH-lu** [reply]
Thanks,
I'll review with that version of Netbox and come back to you.
Can you confirm the status of your IPAM prefixes ?
They should be in status Active and without the tag 'scanrm' for the scan to work.

**Reply by NEO-AMiGA** [reply]
Can confirm that they're active and the one I'm testing with does not have the scanrm tag. 👍

<img width="1469" alt="Screenshot 2024-11-04 at 11 38 40" src="https://github.com/user-attachments/assets/3dbdb69d-5e3d-4b50-8217-9fbd53c15da0">

**Reply by LoH-lu** [reply]
Hey, so I got time to deeply look into this.
So if I understand well you already have IP address in your prefixes, so some of the IP are getting updated and not created.
Do you have a custom field which is mandatory maybe ?
I've added some more logging in netbox_push.py to try catch the issue.

**Comment by NEO-AMiGA** [comment]
Hello. Just checking if you have had the time to test this.

**Comment by LoH-lu** [comment, marked as answer]
@NEO-AMiGA just did a big revamp of the scripts, let me know if you still have issues. (feel free to open new discussion)
